include/NovelRT/Graphics/FontSet.h

```diff
@@ -36,7 +38,7 @@ namespace NovelRT::Graphics {
       }
 
       if (match == _fontCharacters.end()) {
-        throw std::runtime_error("Unable to continue! Unable to return specified character.");
+        throw Exceptions::CharacterNotFoundException("Unable to continue! Unable to return specified character.", c);
```

Comment:
uhh, so I think im being a potato here, but surely the `"Unable to continue! Unable to return specified character."` part should be part of the exception implementation, then we pass it `c` as the only arg?